apps/condo/docs/checklists/pull-request.md

```diff
@@ -0,0 +1,29 @@
+Before you make a pull request
+=====
+
+Apply linter:
+
+```shell
+yarn lint --quiet
+```
+
+Look at errors, reported by TypeScript project-wide in IDE, you are using.
+
+Sync database schema with schemas in Keystone:
+
+```shell
+docker-compose run app yarn workspace @app/condo makemigrations
+```
+
+Sync TypeScript declarations for GraphQL entities:
+
+```shell
+yarn workspace @app/condo maketypes
+```
+
+Check imports, that should be from absolute paths, like following:
+
+```jsx
+import { something } from '@app/condo/domains/…' // correct
```

Comment:
It’s better to move it inside glabal style.md

Reply:
Or move it here

Reply:
It’s better to have one place for style docs

Reply:
Ok, next iteration of working with docs will dedicate for coding style.